From 6dca5f57e066e1fc911da9e51a5019a2ef69401b Mon Sep 17 00:00:00 2001
From: Benjamin Close <Benjamin.Close@clearchain.com>
Date: Wed, 09 Jan 2008 03:40:10 +0000
Subject: Remove reliance on xf86_ansic.h

The file was removed in 1faba797cbfe1a4804b7ea6b47e1ca9d4e4324e4
and prevents compilation.

Found by: Tinderbox
---
diff --git a/src/xf86MagicTouch.c b/src/xf86MagicTouch.c
index d23d89c..12992e6 100644
--- a/src/xf86MagicTouch.c
+++ b/src/xf86MagicTouch.c
@@ -21,9 +21,6 @@
 
 #include <misc.h>
 #include <xf86.h>
-#if !defined(DGUX)
-#include <xf86_ansic.h>
-#endif
 #include <xf86_OSproc.h>
 #include <xf86Xinput.h>
 #include <exevents.h>
--
cgit v0.8.2
